### PR TITLE
Stat rebalance for gun bodies + some new crimes

### PR DESCRIPTION
**Pull request by Fueredoriku**
Rebalanced the gunstats for some bodies:
- Tether and sodie popper no longer have a projectile speed buff
- Damage boost removed from DDR
- Damage boost added to Mower

### CODE REVIEW

File: Assets/Prefabs/UI/Scoreboards.prefab
Comment (toberge) [inline Assets/Prefabs/UI/Scoreboards.prefab]: Irregular

**Review by toberge — APPROVED** [review]
LGTM, will test after it's merged and possibly submit another balancing pr >:)

File: Assets/Prefabs/UI/Scoreboards.prefab
Comment (Fueredoriku) [reply Assets/Prefabs/UI/Scoreboards.prefab]: Fixed 👍

**Pull request merged**
